tox.ini

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 120
+
+[isort]
+force_single_line=True
```

Comment:
can we remove this and use 
`multi_line_output=3` (Vertical Hanging Indent) instead?